lib/puppet/parser/functions/range.rb

```diff
@@ -41,29 +41,9 @@ module Puppet::Parser::Functions
     raise(Puppet::ParseError, "range(): Wrong number of " +
       "arguments given (#{arguments.size} for 1)") if arguments.size < 1
 
-    if arguments.size > 1
-      start = arguments[0]
-      stop  = arguments[1]
-      step  = arguments[2].nil? ? 1 : arguments[2].to_i.abs
-
-      type = '..' # We select simplest type for Range available in Ruby ...
-
-    elsif arguments.size > 0
-      value = arguments[0]
-
-      if m = value.match(/^(\w+)(\.\.\.?|\-)(\w+)$/)
```

Comment:
Why was support for `range('2..3')` and `range('2...3')` and `range('2-3')` removed?